Assets/Scripts/Effect controllers/ReviveVFXController.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.VFX;
+using System.Threading.Tasks;
+
+namespace RogueApeStudio
+{
+    public class ReviveVFXController : MonoBehaviour
+    {
+        //To use this, add a reference to the resurrection VFX to this script.
+        //Reference the Shader Graphs_GlowShader material to this script
+        //Go to the players skinned mesh renderer and add the Shader Graphs_GlowShader material on top of the exiting material.
+        //Reference that same skinned mesh renderer to the VXF property
+
+
+        [SerializeField] private Material _glowMaterial;
+        [SerializeField] private VisualEffect _particle;
+        [SerializeField] private float _minIntensity = 0;
+        [SerializeField] private float _maxIntensity = 1;
+        [SerializeField] private float _opacityIncrement = 0.01f;
+        [SerializeField] private int _loopDelayMiliSeconds = 1;
+        [SerializeField] private int _startDelayMiliSeconds = 0;
+        private Color _defaultColor;
+
+        private void Start()
+        {
+            _defaultColor = _glowMaterial.color;
+            _particle.Play();
+            Glow();
+        }
+
+        private async void Glow()
+        {
+            await Task.Delay(_startDelayMiliSeconds);
+            for (float i = _minIntensity; i < _maxIntensity; i += _opacityIncrement)
+            {
+                IncreaseGlow(i);
+                await Task.Delay(_loopDelayMiliSeconds);
+            }
+
+            for (float i = _maxIntensity; i >= _minIntensity; i -= _opacityIncrement)
+            {
+                DecreaseGlow(i);
+                await Task.Delay(_loopDelayMiliSeconds);
+            }
+        }
```

Comment:
Why does this use basic C# tasks instead of UniTask tasks?

Reply:
Fixed